hail/python/hailtop/batch/docker.py

```diff
@@ -3,13 +3,14 @@
 import os
 from typing import Optional, List
 
-from ..utils import secret_alnum_string, sync_check_shell_output
+from ..utils import secret_alnum_string, sync_check_shell
 
 
 def build_python_image(fullname: str,
                        requirements: Optional[List[str]] = None,
                        python_version: Optional[str] = None,
-                       _tmp_dir: str = '/tmp') -> str:
+                       _tmp_dir: str = '/tmp',
```

Comment:
I have no idea why we named the argument to start with an underscore. That seems like it was a mistake on my part when I first put this functionality in. It could have been a bad renaming operation that did more than I intended. 

Not sure what @danking thinks, but I think I'd be okay with renaming the argument to `tmp_dir` or `tmpdir` and placing a `*` after the renamed `tmp_dir`so that all other arguments after this are keyword argument only. I'd also change the type to Optional[str] and set the default to `/tmp` inside the function.

Sorry about that! Also, why not make the show docker output always True and punt on the tmp_dir problem?

Reply:
You have a better sense of how folks are using this API than I, if you're comfortable with changing tmp_dir, then I am also comfortable changing it. What are you thoughts on making tmp_dir keyword only? Does that feel a step too far?